src/utils/config-defaults.js

```diff
@@ -119,6 +129,9 @@ export default deepFreeze({
   BNavbarToggle: {
     label: 'Toggle navigation'
   },
+  BPagination: {
+    size: 'md'
```

Comment:
This could be set to `null` as there isn't actually a class `b-pagination-md` defined in bootstrap